[IMP] website: "pages" page (#4871)
odoo/documentation: content/applications/websites/website/pages.rst

  - :guilabel:`All`
  - :guilabel:`Signed In`
  - :guilabel:`Restricted Group`: Select the group in the :guilabel:`Authorized group` field.
  - :guilabel:`With Password` : Enter the password in the :guilabel:`Password` field.

.. tip::
   Alternatively, *some* of these properties can also be modified from :menuselection:`Website -->
   Site --> Pages`.

Duplicating pages
~~~~~~~~~~~~~~~~~

To duplicate a page, access the page, then go to :menuselection:`Website --> Site --> Properties`
and click :guilabel:`Duplicate Page`. Enter a :guilabel:`Page Name`, then click :guilabel:`OK`.
By default, the new page is added at the end of the menu, but you can remove it from the menu or
move its position using the menu editor.

.. _website/delete-page:

Deleting pages
~~~~~~~~~~~~~~

To delete a page, proceed as follows:

#. Access the page, then go to :menuselection:`Website --> Site --> Properties` and click
   :guilabel:`Delete Page`.
#. A pop-up window appears on the screen with all links referring to the page you want to delete,
   organized by category. Expand a category, e.g., :guilabel:`Page`, then click on a link to open
   it in a new window.

> #. Once you have updated the links (or set up a :ref:`redirection <website/URL-redirection>`),

Reviewer: You're right! I used your suggestion, but changed it slightly :)

   select the :guilabel:`I am sure about this` check box, then click :guilabel:`OK`.

.. note::
   Visitors are redirected to a 404 error page when they try to access a deleted page, either
   directly or through a link that hasn't been updated. You can prevent this by defining a
   :ref:`redirection <website/URL-redirection>` for the page's URL.

.. _website/URL-redirection:

URL redirects
-------------

Redirecting URLs consists in sending visitors and search engines to a URL that is different from the
one they originally requested. This technique is used, for example, to prevent broken links when you
:ref:`delete a page <website/delete-page>`, :ref:`modify its URL <website/page_properties>`, or move
your site to a new :doc:`domain </administration/maintain/domain_names>`. It can also be used to
improve :doc:`pages/seo`.

To access existing URL redirections for your website and create new ones, :doc:`activate the
developer mode </applications/general/developer_mode>` and go to :menuselection:`Website -->
Configuration --> Redirects`.

> .. note::
>    A record is added automatically everytime you :ref:`modify a page's URL <website/page_properties>`
>    and enable :guilabel:`Redirect Old URL`.

Reviewer 2: You can seperate at any "space". Here's the only suggestion for which I checked the number of characters x) 
```suggestion
.. note::
   A record is added automatically everytime you :ref:`modify a page's URL
   <website/page_properties>` and enable :guilabel:`Redirect Old URL`.
```

To create a new redirection, click the :guilabel:`New` button, then fill in the fields:

- :guilabel:`Name`: Enter a name to identify the redirect.
- :guilabel:`Action`: Select the type of redirection:

   - :guilabel:`404 Not found`: visitors are redirected to a 404 error page when they try to access
     the page.
   - :guilabel:`301 Moved Permanently`: for permanent redirections of static pages; the new URL is
     shown in search engine results and the redirect is cached by browsers.
   - :guilabel:`302 Moved Temporarily`: for short-term redirections, for example, if you are
     redesigning or updating the page. The new URL is neither cached by browsers, nor shown in
     search engine results.
   - :guilabel:`308 Redirect / Rewrite`: for permanent redirections of existing Odoo routes/
     controllers. Use this redirect type to rename a controller, for example, if you wish to rename
     `/shop` into `/market`.

> - :guilabel:`URL from`: Enter the URL to be redirected (e.g., `/about-the-company`) or search for

Reviewer: They are only listed when you select the 308 redirection type. We can have a look together :)

  the desired route/controller and select it from the list.
- :guilabel:`URL to`: For 301, 202 and 308 redirects, enter the URL to be redirected to. If you want
  to redirect to an external URL, make sure to include the protocol (e.g., `https://`).
- :guilabel:`Website`: Select a specific website.
- :guilabel:`Sequence`: to define the order in which redirections are performed, e.g. in the case of
  redirect chains (i.e., a series of redirects where one URL is redirected to another one, which is
  itself further redirected to another URL).

Toggle the :guilabel:`Activate` switch to deactivate the redirection.

.. note::
   - 301 and 308 redirections are cached by browsers.

>    - 404, 301 and 302 redirections only work if the original page has been :ref:`unpublished <website/un-publish-page>`
>      or :ref:`deleted <website/delete-page>`.

Reviewer 2: Same as above :)
```suggestion
   - 404, 301 and 302 redirections only work if the original page has been :ref:`unpublished
     <website/un-publish-page>` or :ref:`deleted <website/delete-page>`.
```

   Site --> Pages`.

Duplicating pages
~~~~~~~~~~~~~~~~~

To duplicate a page, access the page, then go to :menuselection:`Editor --> Site --> Properties` and
click :guilabel:`Duplicate Page`. Enter a :guilabel:`Page Name`, then click :guilabel:`OK`. By
default, the new page is added after the duplicated page in the menu, but you can remove it from the
menu or change its position using the menu editor.

.. _website/delete-page:

Deleting pages
~~~~~~~~~~~~~~

To delete a page, proceed as follows:

#. Access the page, then go to :menuselection:`Editor --> Site --> Properties` and click
   :guilabel:`Delete Page`.
#. A pop-up window appears on the screen with all links referring to the page you want to delete,
   organized by category. To ensure website visitors don't land on a 404 error page, you must update
   all the links on your website referring to the page. To do so, expand a category, e.g.,
   :guilabel:`Page`, then click on a link to open it in a new window.
#. Once you have updated the links (or set up a :ref:`redirection <website/URL-redirection>`),
   select the :guilabel:`I am sure about this` check box, then click :guilabel:`OK`.

> .. note::
>    Visitors are redirected to a 404 error page when they try to access a deleted page, either
>    directly or through a link that hasn't been updated. You can prevent this by defining a
>    :ref:`redirection <website/URL-redirection>` for the page's URL.

Reviewer 3: ```suggestion
```
Not necessary, given it's already covered by points 2. and 3.

.. _website/URL-redirection:

> URL redirects

Reviewer 3: Does this apply to all page types (forum, blog, etc?)
If so, consider moving that part of the documentation to another page or specify this.

-------------

Redirecting URLs consists in sending visitors and search engines to a URL that is different from the
one they originally requested. This technique is used, for example, to prevent broken links when you
:ref:`delete a page <website/delete-page>`, :ref:`modify its URL <website/page_properties>`, or move
your site to a new :doc:`domain </administration/maintain/domain_names>`. It can also be used to
improve :doc:`pages/seo`.

> To access existing URL redirections for your website and create new ones, :doc:`activate the

Reviewer 3: ```suggestion
To access existing URL redirections and create new ones, :doc:`activate the
```

developer mode </applications/general/developer_mode>` and go to :menuselection:`Website -->
Configuration --> Redirects`.

.. note::

>    A record is added automatically everytime you :ref:`modify a page's URL

Reviewer 3: ```suggestion
   A record is added automatically every time you :ref:`modify a page's URL
```

- :guilabel:`Action`: Select the type of redirection:

   - :guilabel:`404 Not found`: visitors are redirected to a 404 error page when they try to access
     the page.
   - :guilabel:`301 Moved Permanently`: for permanent redirections of static pages; the new URL is
     shown in search engine results and the redirect is cached by browsers.
   - :guilabel:`302 Moved Temporarily`: for short-term redirections, for example, if you are
     redesigning or updating the page. The new URL is neither cached by browsers, nor shown in
     search engine results.
   - :guilabel:`308 Redirect / Rewrite`: for permanent redirections of existing Odoo routes/
     controllers. Use this redirect type to rename a controller, for example, if you wish to rename
     `/shop` into `/market`.

- :guilabel:`URL from`: Enter the URL to be redirected (e.g., `/about-the-company`) or search for
  the desired route/controller and select it from the list.
- :guilabel:`URL to`: For 301, 202 and 308 redirects, enter the URL to be redirected to. If you want
  to redirect to an external URL, make sure to include the protocol (e.g., `https://`).
- :guilabel:`Website`: Select a specific website.

> - :guilabel:`Sequence`: to define the order in which redirections are performed, e.g. in the case of

Reviewer 3: ```suggestion
- :guilabel:`Sequence`: to define the order in which redirections are performed, e.g., in the case of
```

  redirect chains (i.e., a series of redirects where one URL is redirected to another one, which is
  itself further redirected to another URL).

Toggle the :guilabel:`Activate` switch to deactivate the redirection.

.. note::
   - 301 and 308 redirections are cached by browsers.
   - 404, 301 and 302 redirections only work if the original page has been :ref:`unpublished
     <website/un-publish-page>` or :ref:`deleted <website/delete-page>`.

.. seealso::

>    - `Redirects and Google Search <https://developers.google.com/search/docs/crawling-indexing/301-redirects>`_

Reviewer 3: ```suggestion
   - `Google documentation on redirects and search <https://developers.google.com/search/docs/crawling-indexing/301-redirects>`_
```
